python/paddle/fluid/tests/unittests/test_detection_map_op.py

```diff
@@ -273,7 +274,7 @@ def init_test_case(self):
 class TestDetectionMAPOpMultiBatch(TestDetectionMAPOp):
     def init_test_case(self):
         super(TestDetectionMAPOpMultiBatch, self).init_test_case()
-        self.class_pos_count = [0, 2, 1]
+        self.class_pos_count = [0, 2, 1, 0]
```

Comment:
Here is a bug before. since num_classes is 4, so the length of this array should be 4.